webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -28,4 +48,146 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
     elif disability_degree >= 30:
         return 620
     elif disability_degree >= 20:
-        return 384
\ No newline at end of file
+        return 384
+    
+    return 0
+    
+    
+class PersonAHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonAHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_a_has_disability') != 'yes':
+            raise ValidationError
+        return values
+        
+class PersonBHasDisabilityPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepPersonBHasDisability.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_disability')
+
+    @root_validator(skip_on_failure=True)
+    def person_b_has_disability(cls, values):
+        if values.get('person_b_has_disability') != 'yes':
+            raise ValidationError
+        return values
+    
+class StepPauschbetrag(LotseFormSteuerlotseStep):
+    pauschbetrag_prefix = 'person_a'
+        
+    def get_overview_value_representation(self, value):
+        result = ''
+        
+        if value == 'yes':
+            result = self.get_pauschbetrag() + ' ' + _('currency.euro')
+            
+        return result
+
+    def get_pauschbetrag(self):
+        return str(calculate_pauschbetrag(
+            has_pflegegrad=self.stored_data.get(self.pauschbetrag_prefix + '_has_pflegegrad', False),            
+            disability_degree=int(self.stored_data.get(self.pauschbetrag_prefix + '_disability_degree', 0)),   
+            has_merkzeichen_bl=self.stored_data.get(self.pauschbetrag_prefix + '_has_merkzeichen_bl', False),
+            has_merkzeichen_tbl=self.stored_data.get(self.pauschbetrag_prefix + '_has_merkzeichen_tbl', False),
+            has_merkzeichen_h=self.stored_data.get(self.pauschbetrag_prefix + '_has_merkzeichen_h', False)
+        ))
```

Comment:
I don't think I like it. I don't like the reflection in this, I have to admit. When I'm changing a field and therefore searching for the field's (full) name, I will not be able to find this - leading to errors. Plus: Setting `pauschbetrag_prefix = 'person_a'` as default is very error-prone (proving here for person b) as you do not *have* to override it.

Reply:
I pulled it back out of the base class because it is too much reflection. I agree with you there. It should be more explicit